Purchases/Purchasing/Purchases.swift

```diff
@@ -1401,7 +1401,8 @@ public extension Purchases {
      * - returns ``RefundRequestStatus``: The status of the refund request.
      * Keep in mind the status could be ``RefundRequestStatus/userCancelled``
      *
-     * If the request was unsuccessful or no active entitlements could be found for the user, an `Error` will be thrown.
+     * If the request was unsuccessful, no active entitlements could be found for the user,
+     * or multiple active entitlements were found for the user, an `Error` will be thrown.
      */
```

Comment:
We can improve formatting by also having a `throws` here, i.e:
```swift
- throws: If the request was unsuccessful, no active entitlements could be found for the user,
* or multiple active entitlements were found for the user, an `Error` will be thrown.
```

In addition, I think we should add a warning or note, something like: 
```swift
- warning: This method should only be used if your user can only have a single active entitlement at a given time. 
* If a user could have more than one entitlement at a time, use ``beginRefundRequest(forEntitlement:)`` instead.
```